src/Command/RefreshOrmCommand.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2017 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\RoutingAutoBundle\Command;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Cmf\Component\RoutingAuto\AutoRouteManager;
+use Symfony\Cmf\Component\RoutingAuto\Mapping\MetadataFactory;
+use Symfony\Cmf\Component\RoutingAuto\UriContextCollection;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class RefreshOrmCommand.
+ *
+ * @author WAM Team <develop@wearemarketing.com>
+ */
+class RefreshOrmCommand extends ContainerAwareCommand
+{
+    /**
+     * @var MetadataFactory
+     */
+    private $metadataFactory;
+
+    /**
+     * @var AutoRouteManager
+     */
+    private $autoRouteManager;
+
+    /**
+     * @var EntityManagerInterface
+     */
+    private $entityManager;
+
+    public function configure()
+    {
+        $this
+            ->setName('cmf:routing:orm:auto:refresh')
```

Comment:
The existing command should handle this (i.e. the existing command should not be coupled to the PHPCR-ODM)

Reply:
hmmm... I will work on that...

Reply:
@dantleech check this and could you tell me if I am doing well?
https://github.com/symfony-cmf/routing-auto-bundle/pull/223/commits/f155b9082624e83a87656ce7243b94525944d08a